README.md

```diff
@@ -57,28 +57,55 @@ For applications wrapping initialization and assignment in an `ObservableObject`
 
 ```swift
 @MainActor
-class AssignmentObserver : ObservableObject {
-    @Published var assignment: String?;
-    var eppoClient: EppoClient = EppoClient(EPPO-SDK-KEY);
+public class AssignmentObserver: ObservableObject {
+    @Published var assignment: String?
+    var eppoClient: EppoClient = EppoClient.configure(apiKey: EPPO_API_KEY)
 
     public init() {
         Task {
             do {
-                try await eppoClient.load();
+                try await eppoClient.loadIfNeeded()
```

Comment:
Safe to call next to each assignment - once a single network request will happen.